Zotero/Scenes/Main/Views/MainViewController.swift

```diff
@@ -177,6 +132,9 @@ extension MainViewController: UISplitViewControllerDelegate {
 
 extension MainViewController: MainCoordinatorDelegate {
     func showItems(for collection: Collection, in library: Library, isInitial: Bool) {
+        if !isInitial {
```

Comment:
Why was it moved from `MasterTopCoordinator` to here? Did it omit some cases?

Reply:
Yes, it omitted the new code paths for restoration, because it is called  directly in the AppCoordinator. 
However, now that preprocessing saves the defaults from restored state, perhaps it is not needed to be moved,
but it seems more correct that way, as previously the deeper `showItems` didn't use at all the `isInitial` argument.